docs/CONTRIBUTING.md

```diff
@@ -69,6 +69,38 @@ systems as they can before sending a pull request.
 *The test suite is run automatically by CI servers when you send a pull
 request.*
 
+Diffing Binaries
+----------------
+
+Binary files are tagged as "binary" in the `.gitattributes` file, but also have
+a `diff=hex` tag, which allows you to see hexdump-style diffs for binaries,
+if you add the following to either your global or repository local git config:
+
+```ini
+[diff "hex"]
+  textconv = hexdump
+  binary = true
+```
+
+Configuring repository local hex diffs:
+
+```sh
+$ git config diff.hex.textconv hexdump
+$ git config diff.hex.binary true
+```
+
+And global, respectively:
+
+```sh
+$ git config --global diff.hex.textconv hexdump
+$ git config --global diff.hex.binary true
+```
+
+If you don't have `hexdump` available on your platform,
+you can try [`hxd`], which is also a bit faster.
+
+[`hxd`]: https://github.com/jhermsmeier/hxd
```

Comment:
Should this link-reference be at the very bottom of the file, along with all the others?